src/Polly.Core/Retry/RetryHelper.cs

```diff
@@ -112,6 +114,7 @@ private static TimeSpan DecorrelatedJitterBackoffV2(int attempt, TimeSpan baseDe
         return TimeSpan.FromTicks((long)Math.Min(formulaIntrinsicValue * RpScalingFactor * targetTicksFirstDelay, maxTimeSpanDouble));
     }
 
+    [SuppressMessage("Style", "IDE0047", Justification = "Parentheses offer less mental gymnastics")]
```

Comment:
We typically suppress with this style:

https://github.com/App-vNext/Polly/blob/b169f88290ee4f3b75092c254215b8343065fa5f/src/Polly.Core/CircuitBreaker/Controller/CircuitStateController.cs#L230-L233

Reply:
done